flocker/provision/_install.py

```diff
@@ -273,39 +279,32 @@ def install_cli_commands_ubuntu(distribution, package_source):
     :param bytes distribution: The distribution the node is running.
     :param PackageSource package_source: The source from which to install the
         package.
+    :param base_url: URL of repository, or ``None`` if we're not using
+        development branch.
 
     :return: a sequence of commands to run on the distribution
     """
-    if package_source.branch:
-        # A development branch has been selected - add its Buildbot repo
-        use_development_branch = True
-        result_path = posixpath.join(
-            '/results/omnibus/', package_source.branch, distribution)
-        base_url = urljoin(package_source.build_server, result_path)
-    else:
-        use_development_branch = False
-
     commands = [
         # Minimal images often have cleared apt caches and are missing
         # packages that are common in a typical release.  These commands
         # ensure that we start from a good base system with the required
         # capabilities, particularly that the add-apt-repository command
         # is available, and HTTPS URLs are supported.
-        sudo_from_args(["apt-get", "update"]),
-        sudo_from_args([
+        run_from_args(["apt-get", "update"]),
```

Comment:
It's nice that this is now consistent between client and node installations.  Previously, the node installation did not have this. It's usually only needed on Docker images.  But it may prevent problems if we Dockerise Flocker.